test/integration/revocation/terraform/variables.tf

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+variable "k8s_test_namespace" {
+  default = "testing"
+}
+
+variable "test_service_account" {
+  default = ""
+}
+
+variable "k8s_vault_connection_address" {}
+
+variable "k8s_config_context" {
+  default = "kind-kind"
+}
+
+variable "k8s_config_path" {
+  default = "~/.kube/config"
+}
+
+variable "k8s_host" {
+  default = "https://kubernetes.default.svc"
+}
+
+variable "k8s_ca_pem" {
```

Comment:
I think the oidc related stuff, along with the k8s_ca_pem is unnecessary for the test.